public/locales/es-ES/translation.json

```diff
@@ -0,0 +1,96 @@
+{
+	"tab.preferences": "Preferencias",
+	"tab.repository": "Repositorio",
+	"tab.tasks": "Tareas",
+	"tab.policies": "Políticas",
+	"tab.snapshots": "Instantáneas",
```

Comment:
Not sure what a good word would be here, but "Instantáneas" does not seem to cut it.